daemons/compute/client-go/main.go

```diff
@@ -139,6 +160,23 @@ func getStatus(ctx context.Context, client pb.DataMoverClient, uid string, maxWa
 	return rsp, nil
 }
 
+func getRequestsList(ctx context.Context, client pb.DataMoverClient, workflow string, namespace string, print bool) []string {
```

Comment:
This function is a little unorthodox - create/status/delete all return the raw protobuf response and an error; and in `func main()` is where the error check / `log.Fatalf` is. Consider following the existing pattern _or_ updating the other routines.

Reply:
The `print bool` variable is unused and can be removed

Reply:
I'll make this consistent with the other functions.

I was originally attempting to reduce duplication by adding in the print statements since I wanted to print out the list before and after deletion and using two functions: `getRequestsList` and `listRequests`. The former to wrap in the error handling/printing and the former to return the raw response.